### PR TITLE
78841d restructure shared components and remove original form scaffold

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR moves the applicant relationship custom page/component out of 1010d and up into the shared ivc directory so it can be used on multiple forms. 

Additionally, lots of original form config for 107959c is removed to make way for the changes coming in the next few PRs.

- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/78848

## Testing done

- Manual
- Unit

## Screenshots

NA, this is a refactor/reorganizational PR

## What areas of the site does it impact?

IVC form apps 10-10d and 10-7959c

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

**Comment by michaelclement** [comment]
Once https://github.com/department-of-veterans-affairs/vets-website/pull/28909 is merged in this will reduce in size significantly

File: src/applications/ivc-champva/10-7959C/config/form.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/config/form.js:1]: Key difference here was just the removal of a lot of the original scaffolding pages that are going to get replaced in the coming days.

File: src/applications/ivc-champva/10-7959C/components/coverages/CoverageField.jsx
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/components/coverages/CoverageField.jsx:1]: Deleted unused component

File: src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js:1]: moved from pages dir to chapters dir

**Review by balexandr — APPROVED** [review]
Lgtm

**Review by reiting — APPROVED** [review]

**Review by cloudmagic80 — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**